portfolio/src/main/webapp/experience.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+  <body style="background c">
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+      
+   <ulsidebar>
+    <li><a href="/index.html">Home</a></li>
+    <li><a href="/introduction.html">About Me</a></li>
+    <li><a class="active" href="/experience.html">Experience</a></li>
+    <li><a href="/academics.html">Academics</a></li>
+    <li><a href="/hidden-talents.html">Hidden Talents</a></li>
+    <li><a href="/photo-gallery.html">Photo Gallery</a></li>
+   </ulsidebar>
```

Comment:
Create a issue for you to later to investigate how not to need to copy this exact block on each page.  You can come back to this when you have time.

Look into options like: 
- html `import` or html `include`. 
- Javascript onload options
- Or even more complex but clean, think about how you can create a custom html element for your `<body>` that could contain the sidebar automatically.